### ISSUE TITLE
Replace mentions of OSFR and OpenSourceFreeRealms to OFR and OpenFreeRealms

### ISSUE THREAD

**Issue by cccfire** [OpenFreeRealms/OpenFreeRealms#6]
### TODO
For the sake of consistency:
- Replace OSFR with OFR
- Replace OpenSourceFreeRealms with OpenFreeRealms

**Comment by GroaxyVRC** [comment]
Or just replace the little amount of things named OFR to OSFR

**Comment by SengokuNadeko** [comment]
yeah a lot of people know us as OSFR instead of OFR so we should just rename any mention of OFR in here to OSFR

**Comment by Gmjjr** [comment]
I agree with Groaxy and Sengoku, it'd be better to make sure we're using OSFR instead of OFR as it's what everyone knows the project as.
